docs/source/conf.py

```diff
@@ -81,7 +83,8 @@
 # built documents.
 #
 # The short X.Y version.
-version = '0.1.0'
+with open(os.path.join('..', 'cellmlmanip', 'version.txt'), 'r') as f:
```

Comment:
I think the documentation has to import the cellmlmanip library elsewhere, so might be better to do the same here?

Reply:
it doesn't import the cellmlmanip library

Reply:
I think Sphinx does in order to generate the docs for it though, from the docstrings.

Reply:
I think it would be better to just `import cellmlmanip; version = release = cellmlmanip.__version__` here

Reply:
This one!

Reply:
ok as @MichaelClerx  and @jonc125 want can you please approve it?